packages/arb-avm-cpp/data_storage/include/data_storage/readwritetransaction.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef data_storage_readwritetransaction_hpp
+#define data_storage_readwritetransaction_hpp
+
+#include <data_storage/readonlytransaction.hpp>
+
+class ReadWriteTransaction : public ReadOnlyTransaction {
```

Comment:
Shouldn't we rename the latter class to ReadTransaction, since a ReadWriteTransaction also extends it?

Reply:
sounds good